index.bs

```diff
@@ -606,6 +701,39 @@ SessionResult = (StatusResult)
 
 ```
 
+<div algorithm>
+
+To <dfn lt="updating the event map">update the event map</dfn>, given a |list of
+event names|, a |list of contexts|, and a |target status|:
```

Comment:
I doesn't affect the normative requirements, but I'm not sure if the *target* in *target status* here refers to a browsing context or realm, or the intended status? I think the latter... Maybe *enable* or *enabled* would work?